### DISCUSSION TITLE
Provide a rich text editor (WYSIWYG/WYSIWYM)

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#820] [Feature Requests]
Flutter doesn't provide one by default. However there is [zefyr](https://github.com/memspace/zefyr), though that project seems to be abandoned and barely receives any updates. We would ideally need to fork it, in order to customize it to our needs.

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by aisbergde** [comment]
I love GitJournal because it is MarkDown based and has no WYSISWG editor. I am fine with the current:
- raw
- markdown
- preview

**Comment by madeindjs** [comment]
This may be related to #157 .

**Comment by aisbergde** [comment]
or some kind of editor as in https://github.com/marktext/marktext
there I can switch between a raw editor and a kind of rich text editor. This could be fine.
But I am fine with #157

**Comment by c-antin** [comment]
Check out https://pub.dev/packages/super_editor

**Comment by vHanda** [comment]
> Check out https://pub.dev/packages/super_editor

I'm hesitant on using it as mobile input is explicitly not on their agenda. My current plan is to experiment with https://github.com/singerdmx/flutter-quill

**Comment by vHanda** [comment]
Note to self -

1. https://github.com/friebetill/delta_markdown
2. https://github.com/ArjanAswal/quill_markdown

Use (1). I think (2) is just a fork of Zefry and doesn't even depend on flutter quill.

**Comment by ashutoshsaboo** [comment]
Is it possible to somehow integrate the lib behind stackedit's markdown editor https://stackedit.io/ ? Not very sure if they vend out a proper easy to integrate lib for their edit yet. It's one of the most user friendly browser-based markdown wysiwyg editors that i've come across. StackOverflow i think also uses the same underlying pagedown editor. @vHanda

**Comment by vHanda** [comment]
> Is it possible to somehow integrate the lib behind stackedit's markdown editor https://stackedit.io/ ? Not very sure if they vend out a proper easy to integrate lib for their edit yet. It's one of the most user friendly browser-based markdown wysiwyg editors that i've come across. StackOverflow i think also uses the same underlying pagedown editor. @vHanda

Sadly it isn't. GitJournal is written in Dart + Flutter. It doesn't use any web-technologies (html / css / javascript).
